packages/backend/src/tests/incomingPayment.ts

```diff
@@ -1,4 +1,3 @@
-import assert from 'assert'
```

Comment:
:thinking: Are we ok keeping `assert` in these `src/tests/*` files?

Reply:
I would argue errors are better here since these functions are just test setup/prerequisites, and not actual test cases where we should be `assert`ing some condition